Build failed on Ubuntu 21.10

Tried to build linux version following https://github.com/Bithack/principia/tree/master/build-linux#readme and it failed.
The problem is likely about missing SDL library, but I'm not sure how to configure it.
```
blue-beaker@blue-beaker-PC:~/G/Git/principia_$  cd /home/blue-beaker/G/Git/principia_/build-linux
blue-beaker@blue-beaker-PC:~/G/Git/principia_/build-linux$ ./autogen.sh
./configure
./go
configure.ac:6: installing './compile'
configure.ac:3: installing './install-sh'
configure.ac:3: installing './missing'
Makefile.am: installing './depcomp'
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for gtk+-2.0 >= 2.0... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: executing depfiles commands
Release: 0
Run:     1
  CC       ../src/tms/core/apparatus2-atlas.o
  CC       ../src/tms/core/apparatus2-backend.o
  CC       ../src/tms/core/apparatus2-batch3d.o
  CC       ../src/tms/core/apparatus2-camera.o
  CC       ../src/tms/core/apparatus2-ddraw.o
  CC       ../src/tms/core/apparatus2-batch.o
  CC       ../src/tms/core/apparatus2-builtin-shaders.inc.o
  CC       ../src/tms/core/apparatus2-entity.o
  CC       ../src/tms/core/apparatus2-event.o
In file included from ../src/tms/core/texture.h:6,
                 from ../src/tms/core/atlas.h:4,
                 from ../src/tms/core/glob.h:1,
                 from ../src/tms/core/batch3d.c:2:
../src/tms/backends/linux/tms/backend/opengl.h:10:10: fatal error: SDL.h: No such file or directory
   10 | #include <SDL.h>
      |          ^~~~~~~
In file included from ../src/tms/core/texture.h:6,
                 from ../src/tms/core/atlas.h:4,
                 from ../src/tms/core/glob.h:1,
                 from ../src/tms/core/atlas.c:1:
../src/tms/backends/linux/tms/backend/opengl.h:10:10: fatal error: SDL.h: No such file or directory

   10 | #include <SDL.h>
      |          ^~~~~~~
compilation terminated.
compilation terminated.
In file included from ../src/tms/core/entity.h:6,
                 from ../src/tms/core/entity.c:6:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: No such file or directory

   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
In file included from ../src/tms/core/ddraw.c:6:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: No such file or directory

   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2658：../src/tms/core/apparatus2-atlas.o] Error 1
make: *** Waiting for tasks to complete....
make: *** [Makefile:2756：../src/tms/core/apparatus2-entity.o] Error 1
make: *** [Makefile:2686：../src/tms/core/apparatus2-batch3d.o] Error 1
make: *** [Makefile:2742：../src/tms/core/apparatus2-ddraw.o] Error 1
In file included from ../src/tms/core/camera.c:5:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: No such file or directory

   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2728：../src/tms/core/apparatus2-camera.o] Error 1
In file included from ../src/tms/core/event.c:6:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: No such file or directory

   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
In file included from ../src/tms/core/texture.h:6,
                 from ../src/tms/core/atlas.h:4,
                 from ../src/tms/core/glob.h:1,
                 from ../src/tms/core/batch.c:1:
../src/tms/backends/linux/tms/backend/opengl.h:10:10: fatal error: SDL.h: No such file or directory

   10 | #include <SDL.h>
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2770：../src/tms/core/apparatus2-event.o] Error 1
make: *** [Makefile:2700：../src/tms/core/apparatus2-batch.o] Error 1
blue-beaker@blue-beaker-PC:~/G/Git/principia_/build-linux$ 
```

> the README inside of the build-linux directory is outdated, look at the build instructions in the regular README file

Still fails with same error log:

```
blue-beaker@blue-beaker-PC:~/G/Git/principia_/build-linux$ ./autogen.sh
blue-beaker@blue-beaker-PC:~/G/Git/principia_/build-linux$ ./configure
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for gtk+-2.0 >= 2.0... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: executing depfiles commands
blue-beaker@blue-beaker-PC:~/G/Git/principia_/build-linux$ ./go
Release: 0
Run:     1
  CC       ../src/tms/core/apparatus2-atlas.o
  CC       ../src/tms/core/apparatus2-batch3d.o
  CC       ../src/tms/core/apparatus2-batch.o
  CC       ../src/tms/core/apparatus2-camera.o
  CC       ../src/tms/core/apparatus2-ddraw.o
  CC       ../src/tms/core/apparatus2-entity.o
  CC       ../src/tms/core/apparatus2-event.o
In file included from ../src/tms/core/ddraw.c:6:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: 没有那个文件或目录
   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2742：../src/tms/core/apparatus2-ddraw.o] 错误 1
make: *** 正在等待未完成的任务....
In file included from ../src/tms/core/camera.c:5:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: 没有那个文件或目录
   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2728：../src/tms/core/apparatus2-camera.o] 错误 1
In file included from ../src/tms/core/texture.h:6,
                 from ../src/tms/core/atlas.h:4,
                 from ../src/tms/core/glob.h:1,
                 from ../src/tms/core/atlas.c:1:
../src/tms/backends/linux/tms/backend/opengl.h:10:10: fatal error: SDL.h: 没有那个文件或目录
   10 | #include <SDL.h>
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2658：../src/tms/core/apparatus2-atlas.o] 错误 1
In file included from ../src/tms/core/entity.h:6,
                 from ../src/tms/core/entity.c:6:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: 没有那个文件或目录
   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
In file included from ../src/tms/core/texture.h:6,
                 from ../src/tms/core/atlas.h:4,
                 from ../src/tms/core/glob.h:1,
                 from ../src/tms/core/batch.c:1:
../src/tms/backends/linux/tms/backend/opengl.h:10:10: fatal error: SDL.h: 没有那个文件或目录
   10 | #include <SDL.h>
      |          ^~~~~~~
compilation terminated.
In file included from ../src/tms/core/texture.h:6,
                 from ../src/tms/core/atlas.h:4,
                 from ../src/tms/core/glob.h:1,
                 from ../src/tms/core/batch3d.c:2:
../src/tms/backends/linux/tms/backend/opengl.h:10:10: fatal error: SDL.h: 没有那个文件或目录
   10 | #include <SDL.h>
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2756：../src/tms/core/apparatus2-entity.o] 错误 1
make: *** [Makefile:2700：../src/tms/core/apparatus2-batch.o] 错误 1
make: *** [Makefile:2686：../src/tms/core/apparatus2-batch3d.o] 错误 1
In file included from ../src/tms/core/event.c:6:
../src/tms/core/tms.h:35:10: fatal error: SDL.h: 没有那个文件或目录
   35 | #include "SDL.h"
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:2770：../src/tms/core/apparatus2-event.o] 错误 1
blue-beaker@blue-beaker-PC:~/G/Git/principia_/build-linux$ 

```

--- a/README.md
+++ b/README.md
@@ -68,7 +68,7 @@ For making Windows release builds you would run the `make_release.sh` script, wh
 
 Install dependencies. For Debian-based distros:
 
-	$ sudo apt-get install automake libgtk2.0-dev libgl-dev libxss-dev libxxf86vm-dev libasound2-dev libudev-dev valgrind libcurl4-openssl-dev libpng-dev libjpeg-dev libfreetype6-dev
+	$ sudo apt-get install automake libgtk2.0-dev libgl-dev libxss-dev libxxf86vm-dev libasound2-dev libudev-dev valgrind libcurl4-openssl-dev libpng-dev libjpeg-dev libfreetype6-dev libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsdl2-mixer-dev libsdl2-gfx-dev
 
 For Arch-based distros:
 

